src/IRrecv.h

```diff
@@ -341,7 +341,11 @@ class IRrecv {
 #endif
 #if DECODE_VESTEL_AC
   bool decodeVestelAC(decode_results *results, uint16_t nbits = kVestelACBits,
-                   bool strict = true);
+                      bool strict = true);
+#endif
+#if DECODE_TCL112AC
+  bool decodeTcl112Ac(decode_results *results, uint16_t nbits = kTcl112AcBits,
```

Comment:
Minor thing: in all other function names AC is written with 2 capital letters, while here as `Ac`

Reply:
Noted. Most new(er) A/Cs added have been 'Ac', at least when I've added them. Trying to balance the legacy usage people might have with trying to keep with the Google style-guide. i.e. First letter of each word (inc abrev. and acronyms) is capitalised only.

I've certainly slipped up in the past. I'll probably try to do a clean up of the names in the 2.6.x release.